### PR TITLE
Fix/stats page -- resolves #271

### LINKED ISSUE #271: [DevEx]: Fix failing stats.spec tests
### Improvement Summary

We should fix all failing automated tests

### Detailed Description

Vue3 updates have broken multiple tests which need to be examined and updated.

Failing test are marked `.skip`.

### PR DESCRIPTION
**Pull request by itsalaidbacklife**
## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
Fixes the features and tests that power the stats page. Resolves #271. Could still use a visual once-over

### CODE REVIEW

File: src/components/StatsLeaderboard.vue [hunk 1]
```diff
@@ -13,14 +13,46 @@
...
+              :top-total-scores="topTotalScores"
+              :season-name="seasonName"
+            />
+          </td>
+        </tr>
+      </tbody>
+    </v-table>
     <!-- https://vuetifyjs.com/en/api/v-data-table/ -->
-    <v-table
+    <!-- <v-data-table
```
Comment (seriouslysean) [inline src/components/StatsLeaderboard.vue:55]: Remove this whole block, we have the git history and don't need it in the template.

File: src/components/StatsLeaderboard.vue [hunk 2]
```diff
@@ -361,13 +392,19 @@ export default {
...
 
 <style scoped>
+th {
+  text-align: left;
+}
 :deep .active-user-stats {
   background-color: rgba(var(--v-theme-accent-lighten3));
 }
-:deep .v-select.fit {
+/* :deep .v-select.fit {
```
Comment (seriouslysean) [inline src/components/StatsLeaderboard.vue:401]: Remove this too

**Review by seriouslysean — APPROVED** [review]
Few small comments, otherwise good to go.

File: src/components/StatsLeaderboard.vue (hunk 1 above)
Comment (itsalaidbacklife) [reply src/components/StatsLeaderboard.vue:55]: Oops 😅

**Pull request merged**